oletools/util/log_helper/_logger_adapter.py

```diff
@@ -0,0 +1,30 @@
+import logging
+from . import _root_logger_wrapper
+
+
+class OletoolsLoggerAdapter(logging.LoggerAdapter):
```

Comment:
Every logger returned by the log_helper [will be wrapped](https://github.com/christian-intra2net/oletools/blob/7cb86312f0a9386926b56e9c8f5578090083edfe/oletools/util/log_helper/log_helper.py#L165) by this class, allowing us to add or change behavior. Since the log_helper is a singleton (an instance is exported in [`__init__.py`](https://github.com/christian-intra2net/oletools/blob/7cb86312f0a9386926b56e9c8f5578090083edfe/oletools/util/log_helper/__init__.py)), if we had used `logging.setLoggerClass()` every logger created would be affected, even if it was created outside of log_helper.